src/Qdrant.Client/Qdrant.Client.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <Title>Qdrant .NET client</Title>
     <Description>.NET client for Qdrant vector database</Description>
-    <TargetFrameworks>net462;netstandard2.0;net6.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(DeveloperBuild)' == 'True'">net6.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
Is there any particular reason to do this, given that we don't rely on any new .NET 8.0 APIs? .NET 8.0 users can still continue using the net6.0 TFM version of this lib just fine etc.

Reply:
(note that the rest of the changes look great - using dotnet SDK 8 and using net8.0 in the tests)

Reply:
the main desire is to run tests on net8.0 and use tooling from net8.0. I'll revert this change but keep the others.